### PR TITLE
EVER-3190/some-task

### PR DESCRIPTION
**Pull request by javierjah**
- change nav active color

### CODE REVIEW

File: src/componentes/nav/nav.scss [hunk 1]
```diff
@@ -11,8 +11,7 @@
       flex: 1;
 
       &.active {
-        // background-color: #dfbaef;
-        background-color: gray;
+        background-color: #dfbaef;
```
Comment (dominicfernandezvergara) [inline src/componentes/nav/nav.scss]: sdfgfg

Comment (dominicfernandezvergara) [reply src/componentes/nav/nav.scss]: pepino

Comment (dominicfernandezvergara) [reply src/componentes/nav/nav.scss]: dfr

Comment (javierjah) [reply src/componentes/nav/nav.scss]: done

**Pull request merged**